rest/api_test.go

```diff
@@ -5023,3 +5023,72 @@ func TestImportOnWriteMigration(t *testing.T) {
 	response = rt.SendAdminRequest("PUT", "/db/doc1?rev=2-44ad6f128a2b1f75d0d0bb49b1fc0019", `{"value":"newer"}`)
 	assertStatus(t, response, http.StatusCreated)
 }
+
+func TestWebhookPropsWithAttachments(t *testing.T) {
```

Comment:
Might be worth moving this test to be just after `TestWebhookSpecialProperties` so it's easier to see them both together if they need to be revisited.

Reply:
Yeah, this test can be moved after TestWebhookSpecialProperties.